typings/rendition.d.ts

```diff
@@ -449,6 +449,7 @@ declare module 'rendition' {
 		// `rowKey` property: the row with a `rowKey` property that matches this
 		// value is highlighted.
 		highlightedRows?: any;
+		getRowClass: (row: T) => string;
```

Comment:
This function should return an array of strings, representing each class name.

Reply:
@LucianBuzzo Ah, so I should move the `join` part inside the render method of the row instead.

Reply:
@dimitrisnl Yep, I think that will be the best approach